__tests__/commands/workspaces.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import {BufferReporter} from '../../src/reporters/index.js';
+import {run as workspace} from '../../src/cli/commands/workspaces.js';
+import * as reporters from '../../src/reporters/index.js';
+import Config from '../../src/config.js';
+import path from 'path';
+
+const fixturesLoc = path.join(__dirname, '..', 'fixtures', 'workspace');
+
+async function runWorkspaces(
+  flags: Object,
+  args: Array<string>,
+  name: string,
+  checkSteps?: ?(config: Config, reporter: BufferReporter) => ?Promise<void>,
+): Promise<void> {
+  const cwd = path.join(fixturesLoc, name);
+  const reporter = new reporters.BufferReporter({stdout: null, stdin: null, isSilent: true});
+
+  try {
+    const config = await Config.create({cwd}, reporter);
+
+    await workspace(config, reporter, flags, args);
+
+    if (checkSteps) {
+      await checkSteps(config, reporter);
+    }
+  } catch (err) {
+    throw new Error(`${err && err.stack}`);
+  }
+}
+
+test('workspaces should list the workspaces', (): Promise<void> => {
+  return runWorkspaces({}, [], 'run-basic', (config, reporter) => {
+    expect(reporter.getBufferJson()).toEqual({
+      'workspace-1': {
+        location: `${path.resolve(fixturesLoc)}/run-basic/packages/workspace-child-1`,
```

Comment:
Assigning `path.resolve(fixturesLoc)` to a variable would make this easier to read.